src/game/GameProcLogIn.cpp

```diff
@@ -120,24 +122,23 @@ void CGameProcLogIn::Init() {
     s_pUIMgr->SetFocusedUI((CN3UIBase *)m_pUILogIn);
 
     // Socket connection..
-    std::string  fsIniFile = (CN3Base::PathGet() / "Server.ini").string();
-    const char * pszIniFile = fsIniFile.c_str();
+    CIni ini(CN3Base::PathGet() / "Server.ini", false);
 
-    int iServerCount = GetPrivateProfileInt("Server", "Count", 0, pszIniFile);
+    int iServerCount = ini.GetInt("Server", "Count", 0);
 
-    char szIPs[256][32]{};
+    std::vector<std::string> vIpAddrs(iServerCount, "");
     for (int i = 0; i < iServerCount; i++) {
-        std::string szKey = std::format("IP{:d}", i);
-        GetPrivateProfileString("Server", szKey.c_str(), "", szIPs[i], sizeof(szIPs[i]), pszIniFile);
+        vIpAddrs[i] = ini.GetString("Server", std::format("IP{:d}", i), "");
     }
-    int iServer = -1;
+
+    std::string szServerIpAddr;
     if (iServerCount > 0) {
-        iServer = rand() % iServerCount;
+        szServerIpAddr = vIpAddrs[rand() % iServerCount];
```

Comment:
Do we need to add  `srand(static_cast<unsigned>(time(0)));` before this line to get a different value every time?